_data/sites.json

```
@@ -12637,6 +12637,16 @@
         ]
     },
 
+    {
+        "name": "Peloton",
+        "url": "https://privacyportal.onetrust.com/webform/18f92a28-d2ae-4a6a-8f99-85b4455e22c0/cfbaf35a-5e06-4010-a6b6-1ad61ba414f1",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
It's kinda customer support/requires human interaction, so I think it should be hard